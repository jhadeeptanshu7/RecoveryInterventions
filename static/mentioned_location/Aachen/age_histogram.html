



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="730a3d57-0b4d-4a2b-b658-6ca5dffed0f3"></div>
            
          
        
      
      
        <script type="application/json" id="348994">
          {"2137b3df-5cdb-4480-b11e-a7976cbac79e":{"roots":{"references":[{"attributes":{},"id":"348911","type":"WheelZoomTool"},{"attributes":{},"id":"348910","type":"PanTool"},{"attributes":{},"id":"348913","type":"SaveTool"},{"attributes":{"overlay":{"id":"348918","type":"BoxAnnotation"}},"id":"348912","type":"BoxZoomTool"},{"attributes":{},"id":"348915","type":"HelpTool"},{"attributes":{},"id":"348914","type":"ResetTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"348910","type":"PanTool"},{"id":"348911","type":"WheelZoomTool"},{"id":"348912","type":"BoxZoomTool"},{"id":"348913","type":"SaveTool"},{"id":"348914","type":"ResetTool"},{"id":"348915","type":"HelpTool"}]},"id":"348916","type":"Toolbar"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"348918","type":"BoxAnnotation"},{"attributes":{},"id":"348936","type":"UnionRenderers"},{"attributes":{},"id":"348935","type":"Selection"},{"attributes":{},"id":"348934","type":"BasicTickFormatter"},{"attributes":{"callback":null,"start":0},"id":"348895","type":"DataRange1d"},{"attributes":{},"id":"348897","type":"CategoricalScale"},{"attributes":{"plot":null,"text":"Aachen - Average Age = 21.0 and Standard Deviation = 0.0"},"id":"348890","type":"Title"},{"attributes":{"below":[{"id":"348901","type":"CategoricalAxis"}],"left":[{"id":"348905","type":"LinearAxis"}],"plot_height":250,"renderers":[{"id":"348901","type":"CategoricalAxis"},{"id":"348904","type":"Grid"},{"id":"348905","type":"LinearAxis"},{"id":"348909","type":"Grid"},{"id":"348918","type":"BoxAnnotation"},{"id":"348928","type":"GlyphRenderer"}],"title":{"id":"348890","type":"Title"},"toolbar":{"id":"348916","type":"Toolbar"},"x_range":{"id":"348893","type":"FactorRange"},"x_scale":{"id":"348897","type":"CategoricalScale"},"y_range":{"id":"348895","type":"DataRange1d"},"y_scale":{"id":"348899","type":"LinearScale"}},"id":"348891","subtype":"Figure","type":"Plot"},{"attributes":{"callback":null,"factors":["21"]},"id":"348893","type":"FactorRange"},{"attributes":{},"id":"348906","type":"BasicTicker"},{"attributes":{},"id":"348899","type":"LinearScale"},{"attributes":{},"id":"348932","type":"CategoricalTickFormatter"},{"attributes":{},"id":"348902","type":"CategoricalTicker"},{"attributes":{"axis_label":"Age","formatter":{"id":"348932","type":"CategoricalTickFormatter"},"major_label_orientation":0.7853981633974483,"plot":{"id":"348891","subtype":"Figure","type":"Plot"},"ticker":{"id":"348902","type":"CategoricalTicker"}},"id":"348901","type":"CategoricalAxis"},{"attributes":{"data_source":{"id":"348925","type":"ColumnDataSource"},"glyph":{"id":"348926","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"348927","type":"VBar"},"selection_glyph":null,"view":{"id":"348929","type":"CDSView"}},"id":"348928","type":"GlyphRenderer"},{"attributes":{"source":{"id":"348925","type":"ColumnDataSource"}},"id":"348929","type":"CDSView"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"348891","subtype":"Figure","type":"Plot"},"ticker":{"id":"348902","type":"CategoricalTicker"}},"id":"348904","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"348934","type":"BasicTickFormatter"},"plot":{"id":"348891","subtype":"Figure","type":"Plot"},"ticker":{"id":"348906","type":"BasicTicker"}},"id":"348905","type":"LinearAxis"},{"attributes":{"callback":null,"data":{"top":[1],"x":["21"]},"selected":{"id":"348935","type":"Selection"},"selection_policy":{"id":"348936","type":"UnionRenderers"}},"id":"348925","type":"ColumnDataSource"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.9},"x":{"field":"x"}},"id":"348926","type":"VBar"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.9},"x":{"field":"x"}},"id":"348927","type":"VBar"},{"attributes":{"dimension":1,"plot":{"id":"348891","subtype":"Figure","type":"Plot"},"ticker":{"id":"348906","type":"BasicTicker"}},"id":"348909","type":"Grid"}],"root_ids":["348891"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('348994').textContent;
                  var render_items = [{"docid":"2137b3df-5cdb-4480-b11e-a7976cbac79e","roots":{"348891":"730a3d57-0b4d-4a2b-b658-6ca5dffed0f3"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>