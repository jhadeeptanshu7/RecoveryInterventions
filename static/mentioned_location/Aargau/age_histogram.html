



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="4f4f7bf3-68e4-4fac-a53a-bcabe6e8609f"></div>
            
          
        
      
      
        <script type="application/json" id="66298">
          {"54f7b27e-626c-492c-b6f9-fec27e1c6e4f":{"roots":{"references":[{"attributes":{},"id":"66240","type":"UnionRenderers"},{"attributes":{},"id":"66214","type":"PanTool"},{"attributes":{},"id":"66215","type":"WheelZoomTool"},{"attributes":{"overlay":{"id":"66222","type":"BoxAnnotation"}},"id":"66216","type":"BoxZoomTool"},{"attributes":{},"id":"66217","type":"SaveTool"},{"attributes":{},"id":"66210","type":"BasicTicker"},{"attributes":{"callback":null,"start":0},"id":"66199","type":"DataRange1d"},{"attributes":{"dimension":1,"plot":{"id":"66195","subtype":"Figure","type":"Plot"},"ticker":{"id":"66210","type":"BasicTicker"}},"id":"66213","type":"Grid"},{"attributes":{"callback":null,"factors":["36"]},"id":"66197","type":"FactorRange"},{"attributes":{"below":[{"id":"66205","type":"CategoricalAxis"}],"left":[{"id":"66209","type":"LinearAxis"}],"plot_height":250,"renderers":[{"id":"66205","type":"CategoricalAxis"},{"id":"66208","type":"Grid"},{"id":"66209","type":"LinearAxis"},{"id":"66213","type":"Grid"},{"id":"66222","type":"BoxAnnotation"},{"id":"66232","type":"GlyphRenderer"}],"title":{"id":"66194","type":"Title"},"toolbar":{"id":"66220","type":"Toolbar"},"x_range":{"id":"66197","type":"FactorRange"},"x_scale":{"id":"66201","type":"CategoricalScale"},"y_range":{"id":"66199","type":"DataRange1d"},"y_scale":{"id":"66203","type":"LinearScale"}},"id":"66195","subtype":"Figure","type":"Plot"},{"attributes":{"plot":null,"text":"Aargau - Average Age = 36.0 and Standard Deviation = 0.0"},"id":"66194","type":"Title"},{"attributes":{},"id":"66218","type":"ResetTool"},{"attributes":{},"id":"66219","type":"HelpTool"},{"attributes":{},"id":"66238","type":"BasicTickFormatter"},{"attributes":{},"id":"66239","type":"Selection"},{"attributes":{},"id":"66236","type":"CategoricalTickFormatter"},{"attributes":{"data_source":{"id":"66229","type":"ColumnDataSource"},"glyph":{"id":"66230","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"66231","type":"VBar"},"selection_glyph":null,"view":{"id":"66233","type":"CDSView"}},"id":"66232","type":"GlyphRenderer"},{"attributes":{"source":{"id":"66229","type":"ColumnDataSource"}},"id":"66233","type":"CDSView"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.9},"x":{"field":"x"}},"id":"66230","type":"VBar"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.9},"x":{"field":"x"}},"id":"66231","type":"VBar"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"66214","type":"PanTool"},{"id":"66215","type":"WheelZoomTool"},{"id":"66216","type":"BoxZoomTool"},{"id":"66217","type":"SaveTool"},{"id":"66218","type":"ResetTool"},{"id":"66219","type":"HelpTool"}]},"id":"66220","type":"Toolbar"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"66238","type":"BasicTickFormatter"},"plot":{"id":"66195","subtype":"Figure","type":"Plot"},"ticker":{"id":"66210","type":"BasicTicker"}},"id":"66209","type":"LinearAxis"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"66195","subtype":"Figure","type":"Plot"},"ticker":{"id":"66206","type":"CategoricalTicker"}},"id":"66208","type":"Grid"},{"attributes":{"callback":null,"data":{"top":[1],"x":["36"]},"selected":{"id":"66239","type":"Selection"},"selection_policy":{"id":"66240","type":"UnionRenderers"}},"id":"66229","type":"ColumnDataSource"},{"attributes":{},"id":"66203","type":"LinearScale"},{"attributes":{},"id":"66201","type":"CategoricalScale"},{"attributes":{},"id":"66206","type":"CategoricalTicker"},{"attributes":{"axis_label":"Age","formatter":{"id":"66236","type":"CategoricalTickFormatter"},"major_label_orientation":0.7853981633974483,"plot":{"id":"66195","subtype":"Figure","type":"Plot"},"ticker":{"id":"66206","type":"CategoricalTicker"}},"id":"66205","type":"CategoricalAxis"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"66222","type":"BoxAnnotation"}],"root_ids":["66195"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('66298').textContent;
                  var render_items = [{"docid":"54f7b27e-626c-492c-b6f9-fec27e1c6e4f","roots":{"66195":"4f4f7bf3-68e4-4fac-a53a-bcabe6e8609f"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>