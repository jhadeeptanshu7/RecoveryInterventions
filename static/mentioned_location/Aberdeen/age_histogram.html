



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="790adf89-2366-4085-8023-0fc1415a3a88"></div>
            
          
        
      
      
        <script type="application/json" id="206770">
          {"2b52bc28-4b6e-4723-81c4-f239d4f39741":{"roots":{"references":[{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"206686","type":"PanTool"},{"id":"206687","type":"WheelZoomTool"},{"id":"206688","type":"BoxZoomTool"},{"id":"206689","type":"SaveTool"},{"id":"206690","type":"ResetTool"},{"id":"206691","type":"HelpTool"}]},"id":"206692","type":"Toolbar"},{"attributes":{"dimension":1,"plot":{"id":"206667","subtype":"Figure","type":"Plot"},"ticker":{"id":"206682","type":"BasicTicker"}},"id":"206685","type":"Grid"},{"attributes":{},"id":"206686","type":"PanTool"},{"attributes":{},"id":"206687","type":"WheelZoomTool"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"206667","subtype":"Figure","type":"Plot"},"ticker":{"id":"206678","type":"CategoricalTicker"}},"id":"206680","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"206710","type":"BasicTickFormatter"},"plot":{"id":"206667","subtype":"Figure","type":"Plot"},"ticker":{"id":"206682","type":"BasicTicker"}},"id":"206681","type":"LinearAxis"},{"attributes":{},"id":"206682","type":"BasicTicker"},{"attributes":{"callback":null,"factors":["19","30"]},"id":"206669","type":"FactorRange"},{"attributes":{"plot":null,"text":"Aberdeen - Average Age = 24.5 and Standard Deviation = 5.5"},"id":"206666","type":"Title"},{"attributes":{},"id":"206712","type":"UnionRenderers"},{"attributes":{"overlay":{"id":"206694","type":"BoxAnnotation"}},"id":"206688","type":"BoxZoomTool"},{"attributes":{},"id":"206689","type":"SaveTool"},{"attributes":{},"id":"206710","type":"BasicTickFormatter"},{"attributes":{},"id":"206711","type":"Selection"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"206694","type":"BoxAnnotation"},{"attributes":{"below":[{"id":"206677","type":"CategoricalAxis"}],"left":[{"id":"206681","type":"LinearAxis"}],"plot_height":250,"renderers":[{"id":"206677","type":"CategoricalAxis"},{"id":"206680","type":"Grid"},{"id":"206681","type":"LinearAxis"},{"id":"206685","type":"Grid"},{"id":"206694","type":"BoxAnnotation"},{"id":"206704","type":"GlyphRenderer"}],"title":{"id":"206666","type":"Title"},"toolbar":{"id":"206692","type":"Toolbar"},"x_range":{"id":"206669","type":"FactorRange"},"x_scale":{"id":"206673","type":"CategoricalScale"},"y_range":{"id":"206671","type":"DataRange1d"},"y_scale":{"id":"206675","type":"LinearScale"}},"id":"206667","subtype":"Figure","type":"Plot"},{"attributes":{"source":{"id":"206701","type":"ColumnDataSource"}},"id":"206705","type":"CDSView"},{"attributes":{"data_source":{"id":"206701","type":"ColumnDataSource"},"glyph":{"id":"206702","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"206703","type":"VBar"},"selection_glyph":null,"view":{"id":"206705","type":"CDSView"}},"id":"206704","type":"GlyphRenderer"},{"attributes":{},"id":"206691","type":"HelpTool"},{"attributes":{},"id":"206690","type":"ResetTool"},{"attributes":{"callback":null,"data":{"top":[1,1],"x":["19","30"]},"selected":{"id":"206711","type":"Selection"},"selection_policy":{"id":"206712","type":"UnionRenderers"}},"id":"206701","type":"ColumnDataSource"},{"attributes":{},"id":"206678","type":"CategoricalTicker"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.9},"x":{"field":"x"}},"id":"206703","type":"VBar"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.9},"x":{"field":"x"}},"id":"206702","type":"VBar"},{"attributes":{},"id":"206675","type":"LinearScale"},{"attributes":{"axis_label":"Age","formatter":{"id":"206708","type":"CategoricalTickFormatter"},"major_label_orientation":0.7853981633974483,"plot":{"id":"206667","subtype":"Figure","type":"Plot"},"ticker":{"id":"206678","type":"CategoricalTicker"}},"id":"206677","type":"CategoricalAxis"},{"attributes":{"callback":null,"start":0},"id":"206671","type":"DataRange1d"},{"attributes":{},"id":"206708","type":"CategoricalTickFormatter"},{"attributes":{},"id":"206673","type":"CategoricalScale"}],"root_ids":["206667"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('206770').textContent;
                  var render_items = [{"docid":"2b52bc28-4b6e-4723-81c4-f239d4f39741","roots":{"206667":"790adf89-2366-4085-8023-0fc1415a3a88"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>