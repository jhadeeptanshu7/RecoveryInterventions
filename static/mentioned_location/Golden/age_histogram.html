



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="fabd2668-b38f-4c7b-af18-fa71f5547b6a"></div>
            
          
        
      
      
        <script type="application/json" id="174502">
          {"7fbae080-3e62-421b-a835-b35c35d6cbbf":{"roots":{"references":[{"attributes":{"callback":null,"data":{"top":[1],"x":["26"]},"selected":{"id":"174443","type":"Selection"},"selection_policy":{"id":"174444","type":"UnionRenderers"}},"id":"174433","type":"ColumnDataSource"},{"attributes":{"source":{"id":"174433","type":"ColumnDataSource"}},"id":"174437","type":"CDSView"},{"attributes":{"data_source":{"id":"174433","type":"ColumnDataSource"},"glyph":{"id":"174434","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"174435","type":"VBar"},"selection_glyph":null,"view":{"id":"174437","type":"CDSView"}},"id":"174436","type":"GlyphRenderer"},{"attributes":{"plot":null,"text":"Golden - Average Age = 26.0 and Standard Deviation = 0.0"},"id":"174398","type":"Title"},{"attributes":{"below":[{"id":"174409","type":"CategoricalAxis"}],"left":[{"id":"174413","type":"LinearAxis"}],"plot_height":250,"renderers":[{"id":"174409","type":"CategoricalAxis"},{"id":"174412","type":"Grid"},{"id":"174413","type":"LinearAxis"},{"id":"174417","type":"Grid"},{"id":"174426","type":"BoxAnnotation"},{"id":"174436","type":"GlyphRenderer"}],"title":{"id":"174398","type":"Title"},"toolbar":{"id":"174424","type":"Toolbar"},"x_range":{"id":"174401","type":"FactorRange"},"x_scale":{"id":"174405","type":"CategoricalScale"},"y_range":{"id":"174403","type":"DataRange1d"},"y_scale":{"id":"174407","type":"LinearScale"}},"id":"174399","subtype":"Figure","type":"Plot"},{"attributes":{},"id":"174419","type":"WheelZoomTool"},{"attributes":{},"id":"174418","type":"PanTool"},{"attributes":{},"id":"174410","type":"CategoricalTicker"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"174442","type":"BasicTickFormatter"},"plot":{"id":"174399","subtype":"Figure","type":"Plot"},"ticker":{"id":"174414","type":"BasicTicker"}},"id":"174413","type":"LinearAxis"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"174399","subtype":"Figure","type":"Plot"},"ticker":{"id":"174410","type":"CategoricalTicker"}},"id":"174412","type":"Grid"},{"attributes":{},"id":"174414","type":"BasicTicker"},{"attributes":{"dimension":1,"plot":{"id":"174399","subtype":"Figure","type":"Plot"},"ticker":{"id":"174414","type":"BasicTicker"}},"id":"174417","type":"Grid"},{"attributes":{},"id":"174421","type":"SaveTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"174418","type":"PanTool"},{"id":"174419","type":"WheelZoomTool"},{"id":"174420","type":"BoxZoomTool"},{"id":"174421","type":"SaveTool"},{"id":"174422","type":"ResetTool"},{"id":"174423","type":"HelpTool"}]},"id":"174424","type":"Toolbar"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"174426","type":"BoxAnnotation"},{"attributes":{"overlay":{"id":"174426","type":"BoxAnnotation"}},"id":"174420","type":"BoxZoomTool"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.9},"x":{"field":"x"}},"id":"174435","type":"VBar"},{"attributes":{},"id":"174422","type":"ResetTool"},{"attributes":{},"id":"174423","type":"HelpTool"},{"attributes":{"fill_color":{"value":"#1f77b4"},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.9},"x":{"field":"x"}},"id":"174434","type":"VBar"},{"attributes":{},"id":"174444","type":"UnionRenderers"},{"attributes":{"axis_label":"Age","formatter":{"id":"174440","type":"CategoricalTickFormatter"},"major_label_orientation":0.7853981633974483,"plot":{"id":"174399","subtype":"Figure","type":"Plot"},"ticker":{"id":"174410","type":"CategoricalTicker"}},"id":"174409","type":"CategoricalAxis"},{"attributes":{},"id":"174442","type":"BasicTickFormatter"},{"attributes":{},"id":"174443","type":"Selection"},{"attributes":{},"id":"174440","type":"CategoricalTickFormatter"},{"attributes":{"callback":null,"start":0},"id":"174403","type":"DataRange1d"},{"attributes":{"callback":null,"factors":["26"]},"id":"174401","type":"FactorRange"},{"attributes":{},"id":"174407","type":"LinearScale"},{"attributes":{},"id":"174405","type":"CategoricalScale"}],"root_ids":["174399"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('174502').textContent;
                  var render_items = [{"docid":"7fbae080-3e62-421b-a835-b35c35d6cbbf","roots":{"174399":"fabd2668-b38f-4c7b-af18-fa71f5547b6a"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>